stock_inventory_quantity_history/tests/test_inventory_quantity_history.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestInventoryQuantityHistory(TransactionCase):
```

Comment:
I think it is better to use @classmethod and add more test cases to also check `inventory_theoretical_qty` when there is a `quantity` for a quant.